de.dlr.sc.virsat.graphiti/src/de/dlr/sc/virsat/graphiti/util/DiagramHelper.java

```diff
@@ -88,9 +93,23 @@ public static StructuralElementInstance getOwningStructuralElementInstance(EObje
 			seiUri = seiUri.appendFileExtension(VirSatProjectCommons.FILENAME_EXTENSION);
 			seiUri = URI.createPlatformResourceURI(seiUri.toFileString(), true);
 			
-			Path path = new Path(resourceUri.toPlatformString(false));
-			IResource iResource = ResourcesPlugin.getWorkspace().getRoot().findMember(path);
-			IProject project = iResource.getProject();
+			
+//			Path path = new Path(resourceUri.toPlatformString(false));
+//			IResource iResource = ResourcesPlugin.getWorkspace().getRoot().findMember(path);
+//			IProject project = iResource.getProject();
+//			ResourceSet resourceSet = VirSatResourceSet.getResourceSet(project);
+//			

```

Comment:
Also here, this comment block should just be deleted.

Reply:
Removed the commented code here as well